### DISCUSSION TITLE
Any ETA or roadmap  for official support with TFC or TFE

### DISCUSSION THREAD

**Discussion by chanakanissanka** [terramate-io/terramate#347] [Q&A] [answered]
Hi there, 
Nice work. out of curiosity does it have any ETA or future road map for Terraform Cloud or Enterprise version official support

**Comment by mariux** [comment, marked as answer]
Hi @chanakanissanka,

Thank you for your question. We discussed this within the team and could not find any blockers on using Terramate in combination with TFE or TFC as of today.

- Terramate generates valid HCL terraform code that can be checked in to the repository and any CI/CD or manual process can pick this up and just execute terraform as normal. 
  If your terraform configuration is using TFE/TFC as a backend or not does not matter to Terramate here.

- Terramate orchestration could help to select a subset of stacks that would need to be run. 
  Here again the TFE/TFC configuration would just picked up by Terraform itself and executed as configured.

- There is no way - that we know of - that Terramate can run inside of TFE/TFC as in being executed remotely. For example you can make tooling like Atlantis aware of Terramate but as in TFE/TFC you do not have this level of control.

Do you have any more specific use case that we missed or should consider to support?
